.kitchen.yml

```diff
@@ -10,8 +10,8 @@ provisioner:
 
 platforms:
   - name: ubuntu-14.04
-  # - name: fedora-20
-  # - name: centos-7.0
+  - name: fedora-20
+  - name: centos-7.0
```

Comment:
probably should add or update to newest OS's (i.e. `ubuntu-16.04`, `fedora-23`, and `centos-7.2`